docs/pages/pmd/rules/java/codestyle.md

```diff
@@ -475,7 +475,7 @@ public class Éléphant {}
 
 To avoid mistakes if we want that a Method, Constructor, Field or Nested class have a default access modifier
 we must add a comment at the beginning of it's declaration.
-By default the comment must be /* default */ or /* package */, if you want another, you have to provide a regular expression.
+By default the comment must be `/* default */` or `/* package */`, if you want another, you have to provide a regular expression.
```

Comment:
This file is generated. We'll need to modify the source of this. It is located here:

https://github.com/pmd/pmd/blob/3a866ddb9fc6f63cf81096a786ff22130d99005d/pmd-java/src/main/resources/category/java/codestyle.xml#L407

We are generating these markdown files during the build from these rulesets.
When you build via `./mvnw verify`, this file will be regenerated and overridden.

Reply:
@adangel Updated.
And reverted the auto-generated file. Hopefully it's the right thing to do.